dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/JarLinkageReport.java

```diff
@@ -92,15 +93,11 @@ public String toString() {
     return builder.toString();
   }
 
-  /** Returns map from the cause of linkage errors to class names affected by the errors. */
-  // TODO this is only used by formatJarLinkageReport in macros.ftl. We should be able to
-  // refactor this to make it a lot clearer by removing ImmutableMultimap from the API.
+  /** 
+   * Map missing classes and members to classes that refer to those items.
+   */
   @Memoized
-  ImmutableMultimap<LinkageErrorCause, String> getCauseToSourceClasses() {
-    return mapCauseToSourceClasses();
-  }
-
-  private ImmutableMultimap<LinkageErrorCause, String> mapCauseToSourceClasses() {
+  ImmutableMultimap<LinkageErrorCause, String> getTargetToSources() {
```

Comment:
The `LinkageErrorCause` isn't the target, right? It's the error (minus the source).

Should this be `getErrorSources()`?